lib/_inspect.js

```diff
@@ -170,7 +170,19 @@ class NodeInspector {
     this.domainNames.forEach((domain) => {
       this[domain] = createAgentProxy(domain, this.client);
     });
+    let startBreakReached = false;
     this.handleDebugEvent = (fullName, params) => {
+      if (!startBreakReached &&
```

Comment:
If this would be inside of the paused event handler (see above), would it be enough to just always resume if the reason is break on start and the environment variable is set?